### DISCUSSION TITLE
Does iPad Pro's LiDAR device improve floor detection?

### DISCUSSION THREAD

**Discussion by suro7** [google/model-viewer#1922] [Q&A]
### Description

Does iPad Pro's LiDAR device improve floor detection? Is there any significant benefit from it?
https://www.apple.com/shop/buy-ipad/ipad-pro

### Version
- model-viewer: v1.2.1

### Browser Affected

- [ ] Chrome
- [ ] Safari

### OS

- [ ] iOS

**Comment by elalish** [comment]
I have no idea; hopefully? It will only affect QuickLook for now, which we have no visibility into.

**Comment by suro7** [comment]
Thanks

**Comment by AndresMercado** [comment]
It does! AR looks so much better on the Ipad Pro. The only issue I am comming across is that there is no indication of loading when the AR button is pressed, it just stays on the same screen while it loads the object for AR behind the scenes, after a few seconds the camera opens and the AR appears right away. An you can occlude the object!

**Comment by elalish** [comment]
@AndresMercado yeah, that annoys me too. Please file the issue with Apple, as I have no way to create a progress bar once their native quick-look app launches.
